data/xp-pen-artist-22r-pro.tablet

```diff
@@ -0,0 +1,27 @@
+# XP-Pen
+# Artist 22R Pro
+
+[Device]
+Name=XP-Pen Artist 22R Pro
+ModelName=
+DeviceMatch=usb|28bd|091b|UGTABLET 21.5 inch PenDisplay Pen;usb|28bd|091b|UGTABLET 21.5 inch PenDisplay Pad
```

Comment:
is this a unique PID? if so we can just leave the name out

Reply:
Ah that's leftover when testing (it turns out the DeviceMatch format changed between master & what I have locally), removed